src/components/30-organisms/modal/README.md

```diff
@@ -28,6 +28,7 @@ import '@axa-ch/modal';
 | `open`    | State of modal visibility (boolean)                    |
 | `forced`  | Disables the internal close mechanisms (boolean)       |
 | `small`   | Smaller version of the modal (boolean, default: false) |
+| `noheader`| Removes the header and padding of the modal root element (boolean, default: false) |
```

Comment:
I'm conflicted about the property name here: the `... and padding` suggests the name is not quite right, but `bare` as in 'bare modal' is also not too suggestive.

If you don't have a clearly superior name, keep it for now.

Reply:
I hope that "Removes the header and the space around the modal content" sounds better